ingest/expression_files/expression_files.py

```diff
@@ -205,7 +205,13 @@ def get_study_expression_file_ids(
         is_raw_count_files = GeneExpression.is_raw_count_file(
             study_id, current_study_file_id, client
         )
-        if is_raw_count_files:
+        if not is_raw_count_files:
```

Comment:
let's flip the order of this if/else so it can be `if is_raw_counts_files` and not need the negation

Reply:
136df7f